bytecode/instr.py

```diff
@@ -34,6 +34,27 @@ def const_key(obj):
         return (type(obj), id(obj))
 
 
+def _pushes_back(opname):
+    if opname in ["CALL_FINALLY"]:
+        return False
+    return (
+        opname.startswith("UNARY_")
+        or opname.startswith("GET_")
+        or opname.startswith("BINARY_")
+        or opname.startswith("INPLACE_")
+        or opname.startswith("BUILD_")
+        or opname.startswith("CALL_")
+    ) or opname in [
+        "LIST_TO_TUPLE",
+        "LIST_EXTEND",
+        "SET_UPDATE",
+        "DICT_UPDATE",
+        "DICT_MERGE",
+        "IS_OP",
+        "CONTAINS_OP",
+    ]
```

Comment:
I believe the following are missing from the list of push back (You could use a tuple rather than a list by the way):
SET_ADD
LIST_APPEND
MAP_ADD
IMPORT_
MAKE_FUNCTION
FORMAT_VALUE

Reply:
it is quite tricky for `IMPORT_` OPs
as `IMPORT_NAME` pops 2 and pushes 1
`IMPORT_FROM` pops 0 and pushes 1
`IMPORT_STAR` simply pops 1 and pushes nothing.

I think it then makes sense to consider only `IMPORT_NAME`

Reply:
I agree for IMPORT_